app/models/result.rb

```diff
@@ -0,0 +1,3 @@
+class Result < ApplicationRecord
+  belong_to :user
```

Comment:
thiếu quan hệ với Question và Option

Reply:
Dạ, 2 cái đấy bọn em k có quan hệ ạ

Reply:
what is `belong_to`?

Reply:
> Dạ, 2 cái đấy bọn em k có quan hệ ạ

vậy thiết kế database có lưu question_id và option_id có tác dụng gì e nhỉ?

Reply:
bọn em đã thêm quan hệ cho question và option với result rồi ạ